client/payment-details/summary/index.tsx

```diff
@@ -487,7 +494,14 @@ const PaymentDetailsSummary: React.FC< PaymentDetailsSummaryProps > = ( {
 					) }
 				</ErrorBoundary>
 			) }
-
+			{ isRefundControlsEnabled &&
+				! _.isEmpty( charge ) &&
+				! charge.order && (
+					<MissingOrderNotice
+						isLoading={ isLoading }
+						formattedAmount={ formattedAmount }
```

Comment:
While testing I noticed that the `formattedAmount` that will be refunded includes the shipping costs. I believe it is expected, but wanted to double check.